base/main_menus/InnMenu.ts

```diff
@@ -0,0 +1,197 @@
+import {YesNoMenu} from "../windows/YesNoMenu";
+import {DialogManager} from "../utils/DialogManager";
+import {GoldenSun} from "../GoldenSun";
+import {TextObj, Window} from "../Window";
+import * as _ from "lodash";
+import {Graphics} from "phaser-ce";
+
+export class InnMenu {
+    public game: Phaser.Game;
+    public data: GoldenSun;
+    public close_callback: Function;
+
+    public choice_menu: YesNoMenu;
+    public dialog_manager: DialogManager;
+    public coin_window: Window;
+    public coin_number: TextObj;
+    public callback: Function;
+    public mask: Graphics;
```

Comment:
You can remove `import {Graphics} from "phaser-ce";` and then do only: `public mask: Phaser.Graphics;`